transactions/src/main/resources/org/jboss/as/txn/subsystem/LocalDescriptions.properties

```diff
@@ -4,43 +4,39 @@ transactions.remove=Removes the transaction subsystem
-transactions.path=Denotes a relative or absolute filesystem path denoting where the transaction manager core should store data. By default the value is treated as relative to the path denoted by the "relative-to" attribute.
-transactions.path.deprecated=Will be removed as this setting has no meaningful effect on the behavior of the transaction manager.
-transactions.relative-to=References a global path configuration in the domain model, defaulting to the JBoss Application Server data directory (jboss.server.data.dir). The value of the "path" attribute will treated as relative to this path. Use an empty string to disable the default behavior and force the value of the "path" attribute to be treated as an absolute path.
-transactions.relative-to.deprecated=Will be removed as this setting has no meaningful effect on the behavior of the transaction manager.
-transactions.jts=If true this enables the Java Transaction Service.
-transactions.use-journal-store=Use the journal store for writing transaction logs. Set to true to enable and to false to use the default log store type. The default log store is normally one file system file per transaction log. The server should be restarted for this setting to take effect. It's alternative to jdbc based store.
-transactions.journal-store-enable-async-io=Whether AsyncIO should be enabled for the journal store. Default is false. The server should be restarted for this setting to take effect.
-transactions.use-hornetq-store=Use the hornetq store for writing transaction logs. Set to true to enable and to false to use the default log store type. The default log store is normally one file system file per transaction log. The server should be restarted for this setting to take effect. It's alternative to jdbc based store.
+transactions.jts=If true this enables the Java Transaction Service. Use of the JTS needs configuration in IIOP OpenJDK where Transactions parameter needs to be set to full.
+transactions.use-journal-store=Use the journal store for writing transaction logs. Set to true to enable and to false to use the default log store type. The default log store creates normally one file file per transaction log. The journal one consists from one file for all the transactions. It's alternative to jdbc based store.
+transactions.journal-store-enable-async-io=Whether AsyncIO should be enabled for the journal store. Default is false. For this settings being active journal natives libraries needs to be available.
+transactions.use-hornetq-store=Use the hornetq store for writing transaction logs. Set to true to enable and to false to use the default log store type. The default log store is normally one file system file per transaction log.It's alternative to jdbc based store.
```

Comment:
the use-hornetq-store and hornetq-store-enable-async-io have been deprecated. anyway, I replace the hornetq with "journal"